app/code/Magento/CmsGraphQl/composer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "magento/module-cms-graph-ql",
+    "description": "N/A",
+    "type": "magento2-module",
+    "require": {
+        "php": "~7.1.3||~7.2.0",
+        "magento/framework": "*"
+    },
+    "suggest": {
+        "magento/module-graph-ql": "*",
```

Comment:
Should we have dependency on CMS module (or at least `suggest`, if there are no hard dependencies)?

Reply:
Hi
I guess Travis will fail in this case with the "redundant dependency" error.